ushahidi/platform-client
# UI-fixes

This pull request makes the following changes:
- fixes error-labels in targeted surveys
- fixes error-labels-app-wide
- adds pl-version v3.7.2-rc.35

Testing checklist:
- go to create targeted survey
- Add a name and remove it
- [x] an error-label should appear and not affect field below it
- Add a description and remove it
- [x] an error-label should appear and not affect field below it
- click on continue
- [x] error-labels should appear and not affect field below it
- [x] you should not move to the next section
- Add name and description, click continue
- Click continue again
- [x] error-labels should appear and not affect field below it
- [x] you should not move to the next section
- Add a country
- Click on continue
- [x] an error-label should appear above numbers and not affect any fields around it
- Add a number and a faulty number
- [x] an error-label should appear above faulty numbers and not affect any fields around it
- fix the number and click on continue
-click on continue again
- [x] an error-label should appear above numbers and not affect any fields around it
- [x] you should not move to the next section
- click on 'add question'
- click on add again
- [x] an error-label should appear and not affect any fields around it
- [x] You should still be in the modal
- Add a question and click save
- Add the same question again
- [x] an error-label should appear and not affect any fields around it
- [ ] You should still be in the modal
- Add a different question in the box and save
- Click on continue
- You should be moved to the next section

I have also changed the html-syntax app-wide to display the new error-labels correctly. Test for example in post-edit, post add and add new user, error-labels should appear and not affect fields around it. 

The colours look a bit different than before, this is correct and is part of the adjustment for accessibility.

Known-error: error-labels are still affecting fields around them in settings-general and the user-list in settings in wrong. This will be fixed in next PL-tag.
 


Fixes ushahidi/platform# .

Ping @ushahidi/platform
